lib/auth/bento-theme/login/template.ftl

```diff
@@ -86,6 +81,11 @@
     x-id="['language-select']"
 >
   <div class="pf-v5-c-login__container">
+    <div id="kc-header" class="${properties.kcHeaderClass!}">
```

Comment:
this renders weirdly to me:
![image](https://github.com/user-attachments/assets/f23eb918-6a26-4ace-889a-9fe297e0d2d9)